crates/types/src/config/object_store.rs

```diff
@@ -18,35 +18,41 @@ use serde_with::serde_as;
 #[serde(rename_all = "kebab-case")]
 #[builder(default)]
 pub struct ObjectStoreOptions {
-    /// The AWS configuration profile to use for S3 object store destinations.
+    /// The AWS configuration profile to use for S3 object store destinations. If you use
+    /// named profiles in your AWS configuration, you can replace all the other settings with
+    /// a single profile reference. See the [AWS documentation on profiles]
+    /// (https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html) for more.
```

Comment:
This only works for AWS, right? Can we make it clear that this is not applicable for other S3-compatible object stores?

Reply:
No, this works with all S3 protocol stores. Would you like me to make this more explicit? I didn't think it was needed as phrased. For example I have this in my `~/.aws/config`:

```
[profile minio]
region = us-east-1
endpoint_url = http://localhost:9000
aws_access_key_id = minioadmin
aws_secret_access_key = minioadmin
```